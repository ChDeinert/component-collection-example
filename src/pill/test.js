import React from 'react';
import renderer from 'react-test-renderer';
import Pill from './';

it('renders correctly', () => {
  const renderedTree = renderer.create(<Pill>Content</Pill>);
  expect(renderedTree.toJSON()).toMatchSnapshot();
});
