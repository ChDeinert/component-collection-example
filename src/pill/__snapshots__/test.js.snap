// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li
  style={
    Object {
      "display": "block",
      "float": "left",
      "fontSize": "14px",
      "lineHeight": "20px",
      "listStyleType": "none",
      "marginLeft": "2px",
      "position": "relative",
    }
  }
>
  <a
    href="#"
    onClick={undefined}
    style={
      Object {
        "border": "1px solid rgb(200, 200, 200)",
        "borderBottomLeftRadius": "4px",
        "borderBottomRightRadius": "4px",
        "borderTopLeftRadius": "4px",
        "borderTopRightRadius": "4px",
        "boxSizing": "borderBox",
        "color": "rgb(51, 122, 183)",
        "cursor": "auto",
        "display": "block",
        "fontFamily": "\\"Helvetica Neue\\", Helvetica, Arial, sansSerif",
        "fontSize": "14px",
        "lineHeight": "20px",
        "listStyleImage": "none",
        "listStylePosition": "outside",
        "listStyleType": "none",
        "paddingBottom": "10px",
        "paddingLeft": "15px",
        "paddingRight": "15px",
        "paddingTop": "10px",
        "position": "relative",
        "textAlign": "left",
        "textDecorationColor": "rgb(51, 122, 183)",
        "textDecorationLine": "none",
        "textDecorationStyle": "solid",
        "textSizeAdjust": "100%",
      }
    }
  >
    Content
  </a>
</li>
`;
